File: pgm/ABC_Visual_Basic/ABC_133_A.vb
' Imports System.Collections.Generic
' Imports System.Globalization
' Imports System.IO
' Imports System.Linq
' Imports System.Text
' Imports System.Text.RegularExpressions

Class Main
    Public Shared Sub Main()
        Dim N, A, B As Integer
        Dim arr() As String
        arr = Split(Console.ReadLine())
        N = arr(0) : A = arr(1) : B = arr(2)
        Console.WriteLine(Math.min(N * A, B))
        Return
    End Sub
End Class

File: pgm/ABC_C#/ABC_133_A.cs
using System;
// using System.Collections.Generic;
// using System.Globalization;
// using System.IO;
// using System.Linq;
// using System.Text;
// using System.Text.RegularExpressions

class Program
{
    static void Main()
    {
        string[] arr = Console.ReadLine().Split();
        int N, A, B;
        N = int.Parse(arr[0]);
        A = int.Parse(arr[1]);
        B = int.Parse(arr[2]);
        Console.WriteLine(Math.Min(A * N, B));
        return;
    }
}